modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/db/Manifest.scala

```diff
@@ -31,40 +28,49 @@ object Manifest {
     "shredded_cardinality"
   )
 
-  def initialize[F[_]: MonadThrow: Logging: Timer: AWS, C[_]: DAO: Monad](target: StorageTarget)
-                                                                         (implicit F: Transaction[F, C]): F[Unit] = {
-    F.transact(setup[C](target.schema)).attempt.flatMap {
+  def initialize[F[_]: MonadThrow: Logging: Timer: AWS, C[_]: DAO: Monad](
+    target: StorageTarget
+  )(implicit F: Transaction[F, C]): F[Unit] =
+    F.transact(setup[C](target.schema, target)).attempt.flatMap {
       case Right(InitStatus.Created) =>
         Logging[F].info("The manifest table has been created")
       case Right(InitStatus.Migrated) =>
-        Logging[F].info(s"The new manifest table has been created, legacy 0.1.0 manifest can be found at $LegacyName and can be deleted manually")
+        Logging[F].info(
+          s"The new manifest table has been created, legacy 0.1.0 manifest can be found at $LegacyName and can be deleted manually"
+        )
       case Right(InitStatus.NoChanges) =>
         Monad[F].unit
       case Left(error) =>
         Logging[F].error(error)("Fatal error has happened during manifest table initialization") *>
           MonadError[F, Throwable].raiseError(new IllegalStateException(error.toString))
     }
-  }
 
-  def setup[F[_]: Monad: DAO](schema: String): F[InitStatus] =
-    for {
-      exists <- Control.tableExists[F](Name)
-      status <- if (exists) for {
-        columns <- Control.getColumns[F](Name)
-        legacy = columns.toSet === LegacyColumns.toSet
-        status <- if (legacy)
-          Control.renameTable[F](Name, LegacyName) *>
-            create[F].as[InitStatus](InitStatus.Migrated)
-        else
-          Monad[F].pure[InitStatus](InitStatus.NoChanges)
-      } yield status else create[F].as(InitStatus.Created)
-      _ <- status match {
-        case InitStatus.Migrated | InitStatus.Created =>
-          DAO[F].executeUpdate(Statement.CommentOn(s"$schema.$Name", "0.2.0"))
-        case _ =>
-          Monad[F].unit
-      }
-    } yield status
+  def setup[F[_]: Monad: DAO](schema: String, target: StorageTarget): F[InitStatus] = target match {
+    case _: Databricks => create[F].as(InitStatus.Created)
+    case _ =>
+      for {
+        exists <- Control.tableExists[F](Name)
+        status <- if (exists) for {
+          columns <- Control.getColumns[F](Name)
+          legacy = columns.toSet === LegacyColumns.toSet
+          status <- if (legacy)
+            Control.renameTable[F](Name, LegacyName) *>
+              create[F].as[InitStatus](InitStatus.Migrated)
+          else
+            Monad[F].pure[InitStatus](InitStatus.NoChanges)
+        } yield status
+        else create[F].as(InitStatus.Created)
+        _ <- status match {
+          case InitStatus.Migrated | InitStatus.Created =>
+            target match {
+              case _: Redshift => DAO[F].executeUpdate(Statement.CommentOn(s"$schema.$Name", "0.2.0"))
```

Comment:
Looks like this changes it to not comment on Snowflake (Redshift only).  Is this deliberate?

Reply:
It is an accident. But comment(or lack thereof) on the manifest makes zero difference for the snowflake, which doesn't have manifest versions.